### ISSUE TITLE
Fix the build for `main` branch

### ISSUE THREAD

**Issue by ruspl-afed** [eclipse-mylyn/org.eclipse.mylyn#10]
Let's try to fix the build for pre-merged version of mylyn sources

**Comment by ruspl-afed** [comment]
@wimjongman @BeckerFrank  please have a look at build results https://ci.eclipse.org/mylyn/job/github-mylyn/job/PR-11/
It may require further cleanup and tuning of deploy, but in general it looks good to me.
WDYT?

**Comment by wimjongman** [comment]
That was quick!

**Comment by BeckerFrank** [comment]
Is there anything left here or can we close this issue?

**Comment by ruspl-afed** [comment]
We have the only point before we can claim the good shape: restore stable and release build process. But this could go to another ticket and this one can be closed.

**Comment by wimjongman** [comment]
> We have the only point before we can claim the good shape: restore stable and release build process. 

I don't understand what you mean by that. Can you explain?

**Comment by wimjongman** [comment]
Yes, we should. Frank is under the weather but maybe next week Wednesday 14:00 to 15:00?

**Comment by ruspl-afed** [comment]
next Wednesday looks good for me, but I need to understand the time zone of your proposed 14:00 to 15:00

**Comment by wimjongman** [comment]
Yep. Sorry. I will send an invite to you and Frank.
